develop.html.mako

```diff
@@ -112,6 +113,26 @@ requested changes are not responded towards will be closed.</li>
 <li>Please make sure you are configured to run the current <a href="#environment">Development Environment</a>
     requirements, which will help ensure your PRs pass automatic tests.</li>
 
+<li>Instead of editing the Changelog file itself, please adopt the following workflow:
+    <ul>
+        <li>Create a per-issue changelog file with the path
+            `/doc/build/changelog/unreleased_{Version}/{IssueId}.rst`, in which
+            `IssueId` is the auto-generated GitHub Issue Id.
+            </li>
+        <li>If the PR does not have a corresponding issue, use a placeholder file
```

Comment:
any PR that warrants a changelog entry would need to have an issue number.    I very rarely make changelog-worthy changes without creating an issue and outside contributors shouldn't be doing it ever.